### PR TITLE
Update/balance shows undefined

### LINKED ISSUE #139: Delegate Vote page shows `undefined` while data is loading
If Delegate page/component is mounted while DAO data (specifically, token data) is loading, `undefined` is shown as the token balance. This is ugly and there should be a more user-friendly display.

### PR DESCRIPTION
**Pull request by Da-Colon**
Closes #139 
## Overview

Adds a DataLoaderWrapper Component to wrap single Elements (or string) that might need load before displaying in UI

## Screenshots
Can't really get a good look unless you pull down branch. 
In `Delegate.tsx` line 62, replace props `isLoading={!userBalance && !symbol}>{`${userBalance}` replace with just `isLoading` to see it in action

### CODE REVIEW

**Comment by adamgall** [comment]
@Da-Colon is this PR for issue #139? any others?

**Review by adamgall — CHANGES_REQUESTED** [review]

**Comment by Da-Colon** [comment]
@adamgall Wrapper is added to `Current Delegatee`

File: src/components/DAOs/DAO/Delegate.tsx [hunk 1]
```diff
@@ -56,12 +57,17 @@ function Delegate() {
...
+            </span>
           </div>
           <div className="flex mx-2 my-1 text-gray-50">
-            Current Delegatee: &nbsp;
+            Current Delegatee:{" "}
             <EtherscanLink address={delegatee}>
-              <p className="text-gold-500">{delegateeDisplayName}</p>
+              <span className="text-gold-500 ml-2">
+                <DataLoadingWrapper isLoading={!delegateeDisplayName}>{delegateeDisplayName}</DataLoadingWrapper>
+              </span>
```
[the remark this answers is not in the document]
Comment (Da-Colon) [reply src/components/DAOs/DAO/Delegate.tsx:70]: Let me try and recreate that.

[the remark this answers is not in the document]
Comment (Da-Colon) [reply src/components/DAOs/DAO/Delegate.tsx:70]: yeah there were some style updates on master. updating now.

**Comment by Da-Colon** [comment]
@adamgall should be good to go
image

**Review by adamgall — APPROVED** [review]
Aw yeah flip those components

**Pull request merged**